inst/extdata/example_params.toml

```diff
@@ -7,14 +7,17 @@ r_prior_sd = 1
 sigma_rt_prior = 0.1
 dur_inf = 7
 
-sigma_i0_prior_mode = 0
-sigma_i0_prior_sd = 0.5
-i0_certainty = 5
+sigma_i_first_obs_prior_mode = 0
```

Comment:
Just a general comment that this `i_first_obs` is more explicit, but I wonder if we could just keep the `i0` as its a bit less verbose, and then make sure we are more clear in the model definition the `i0` or $I(t=0)$ is the start of the observations?